LPS-89071 Fix user context provider - if portal session is not present fall back

matethurzo opened the pull request.

Seiphon committed:
LPS-88851 Handle fragment module in WatchTask

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to site-navigation-admin-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to message-boards-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to hello-soy-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to fragment-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to layout-taglib

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-brightness

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-contrast

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-crop

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-effects

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-resize

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-rotate

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-image-editor-capability-saturation

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to wiki-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to portlet-configuration-css-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to layout-admin-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to journal-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to hello-soy-navigation-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to adaptive-media-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to flags-taglib

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to asset-list-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to asset-categories-selector-web

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-taglib

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to fragment-display-web

Commit:
LPS-87616 Add methods to save and delete permission for data record collection

Commit:
LPS-87616 Implement new methods

Commit:
LPS-87616 Add new exception classes

Commit:
LPS-87616 Add source formatter exclusion

Commit:
LPS-87616 Update GraphQL schema

Commit:
LPS-87616 Move code to base class

Commit:
LPS-87616 Add GraphQL model classes

Commit:
LPS-87616 Add GraphQL data fetchers

Commit:
LPS-87616 Wire data fetchers

Commit:
LPS-87616 Add language term

Commit:
LPS-87616 buildLang

Commit:
LPS-87616 Change language key

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Commit:
LRQA-45896 Add type attribute to get-java-jdk-home macro

Commit:
LRQA-45896 Add get-java-jdk-bundle-type macro

Commit:
LRQA-45896 Execute get-java-jdk-home macro with JDK attribute

Commit:
LRQA-45896 Add additional Java JDK batches

Commit:
LRQA-45896 Update Java JDK bundle versions

wcao20170619 committed:
LPS-89318 portal-search-elasticsearch6-impl: Test: Setters, not protected fields; eliminate IndexCreator constructors and subclasses

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 portal-search-elasticsearch6-impl: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

arboliveira committed:
LPS-89318 portal-search-elasticsearch6-impl: Test: Fix anonymous classes causing directory name to become an empty string

https://issues.liferay.com/browse/LPS-89318

victorg1991 committed:
APIO-311 Add a new method in the Mapper interface to add the embedded action target

victorg1991 committed:
APIO-311 Implement new mapper method in JSONLD mapper

victorg1991 committed:
APIO-311 Map embedded action url

victorg1991 committed:
APIO-311 Update tests

ruben-pulido committed:
APIO-238 Create filter to remove Accept-Language header if its value is empty to prevent 500 error from happening

Co-authored-by: Víctor Galán <victor.galan@liferay.com>

ruben-pulido committed:
APIO-238 Unit tests

Co-authored-by: Víctor Galán <victor.galan@liferay.com>

ruben-pulido committed:
APIO-238 Integration tests

Co-authored-by: Víctor Galán <victor.galan@liferay.com>

ngaskill committed:
LRDOCS-6136 Javadoc formatter

wcao20170619 committed:
LPS-89318 portal-search-solr7-impl: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 portal-search-solr7-impl: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

ngaskill committed:
LRDOCS-6136 Javadoc fix in FieldMode

wcao20170619 committed:
LPS-89318 portal-search-test: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 journal-test-util: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 journal-test: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 document-library-test-util: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 document-library-test: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

javiergamarra committed:
LRDOCS-6136 close ul

javiergamarra committed:
APIO-323 naive sort

Commit:
LRQA-45896 Default to java.jdk.architecture in architecture attribute

Commit:
LRQA-45896 Change default java.jdk.architecture to x64

alejandrohdezma committed:
APIO-323 Avoid using star import

alejandrohdezma committed:
APIO-323 SF

Seiphon committed:
LPS-88851 SF

ruben-pulido committed:
APIO-238 Add ruben-pulido to subrepo.merge.pull.mention.users

bryceosterhaus committed:
LRQA-44612 - Use liferay-npm-scripts@v1.2.0 for updated test reporter

pavel-savinov committed:
LPS-89154 Redirect user to the content page editor to edit layout associated with the page template entry

pavel-savinov committed:
LPS-89154 Use content page editor to edit asset display pages

pavel-savinov committed:
LPS-89154 Edit layout page templates and display pages using existing content pages editor

brianikim committed:
LPS-89503 Returned result needs to respect pagination

wcao20170619 committed:
LPS-89318 asset-test: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 asset-categories-test: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-89318 users-admin-test-util: Test: Setters, not protected fields

https://issues.liferay.com/browse/LPS-89318

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

Commit:
LRQA-45896 Add batches to environment suite

jkappler committed:
LPS-89154 Don't show icons when in template view

bryceosterhaus committed:
LPS-88955 - Apply liferay-npm-scripts to frontend-taglib-clay

bryceosterhaus committed:
LRQA-44612 - Update liferay-jest-junit-reporter dependency

Commit:
LRQA-45896 Simplify

marcelabc committed:
LPS-87616 Update guest unsupported permissions

marcelabc committed:
LPS-87616 Make the parameter mandatory on save permissions request for DEDataDefinition

marcelabc committed:
LPS-87616 Update the save model permissions request for DEDataDefinition

marcelabc committed:
LPS-87616 Update the save permissions request/response for DEDataRecordCollection

marcelabc committed:
LPS-87616 Update the save model permissions request/response for DEDataRecordCollection

marcelabc committed:
LPS-87616 Update GraphQL schema

marcelabc committed:
LPS-87616 Add GraphQL data fetchers

marcelabc committed:
LPS-87616 JavaDocs

marcelabc committed:
LPS-87616 Add new exceptions

thektan committed:
LPS-88934 Support multiple entity selection

marcelabc committed:
LPS-87616 Add integration tests

marcelabc committed:
LPS-87616 Update site member default permissions

Commit:
LPS-89375 Add settings form to Element Sets

Commit:
LPS-89375 Fix javascript loading order issues

Commit:
LPS-89375 Rename for consistency

gamerson committed:
LPS-88851 protect against symbolicNames without '('

gamerson committed:
LPS-88851 simplify

gamerson committed:
LPS-88851 extract common private method

gamerson committed:
LPS-88851 SF

Preston-Crary committed:
LPS-89878 Export common jackson packages from the system.

Preston-Crary committed:
LPS-89878 Share the jars instead of copying them.

thektan committed:
LPS-88934 Use liferay-item-selector-dialog module

Preston-Crary committed:
LPS-89878 Remove unused JSONValidator support.

Preston-Crary committed:
LPS-89878 Remove conflicting com-github-fge-json-schema-validator library.

Preston-Crary committed:
LPS-89878 Remove patches for com-github-fge-json-schema-validator.

Preston-Crary committed:
LPS-89878 Update projects.

thektan committed:
LPS-88934 Split TypedInput component into separate components

ealonso committed:
LPS-89154 Creates new module to encapsulate all code related with layout content editor

ealonso committed:
LPS-89154 Moves code from type controller, no logic changes

ealonso committed:
LPS-89154 Language keys

ealonso committed:
LPS-89154 Rename class

ealonso committed:
LPS-89154 Moves web keys

ealonso committed:
LPS-89154 Adds new API module

ealonso committed:
LPS-89154 Moves portlet keys to API module

ealonso committed:
LPS-89154 packageinfo

ealonso committed:
LPS-89154 Uses new portlet

JosemariaMunoz committed:
LPS-89632 If exist categoryId as request parameter, add categoryId to portletURL

ealonso committed:
LPS-89154 Clean build.gradle

ealonso committed:
LPS-89154 No needed

ealonso committed:
LPS-89154 Moves soy and js for fragmenst editor to content page editor portlet

ealonso committed:
LPS-89154 Moves css for fragment editor to content page editor

ealonso committed:
LPS-89154 Update paths

ealonso committed:
LPS-89154 js files

ealonso committed:
LPS-89154 Auto

GergoMathe committed:
LPS-89660 Revert applying PermissionCheckerTestRule to affected tests as we'll solve it at the CT side

ealonso committed:
LPS-89154 Uses npmResolvedPackageName

GergoMathe committed:
LPS-89660 Use CompanyThreadLocal in CTManager if the user can't be obtained

ealonso committed:
LPS-89154 Copy file from layout admin web, no logic changes

GergoMathe committed:
LPS-89660 Log a warning only, with correct message

ealonso committed:
LPS-89154 Removes fragment display web portlet

ealonso committed:
LPS-89154 Rename to main.scss

pavel-savinov committed:
LPS-89154 All the logic is encapsulated in the fragments editor module, no need to export list and sidebar separately

ealonso committed:
LPS-89154 Adds new class for constants

ealonso committed:
LPS-89154 Check for layoutPageTemplateEntry

ealonso committed:
LPS-89154 Match with ContentLayoutTypeController

ealonso committed:
LPS-89154 Rename to ContentPageLayoutEditor

NemethNorbert committed:
LPS-89615 Update ComboServlet with .min.js and .min.css suffix

ealonso committed:
LPS-89154 Moves FragmensEditorDisplayContext, no logic changes

ealonso committed:
LPS-89154 Moves common constant to API module

ealonso committed:
LPS-89154 Adds dependencies

ealonso committed:
LPS-89154 Removes no needed methods

thektan committed:
LPS-88934 Update language properties path for webpack dev server

Preston-Crary committed:
LPS-89878 Versioning.

Commit:
LRQA-45896 Change default Java JDK type to Oracle

marcelabc committed:
LPS-87616 Remove GraphQL of Data Record Collection permissions

Preston-Crary committed:
LPS-89878 Build lib versions.

thektan committed:
LPS-88934 Create a new group for multiple selected items

brianikim committed:
LPS-89869 Blacklisted chars should be unescaped for accurate comparison against folder name

patriciajeanperez committed:
LRQA-45606 Expand Versioning panel

petershin committed:
LPS-89415 Remove unsupported HTTP methods

petershin committed:
LPS-89415 Remove the schema name from the path

petershin committed:
LPS-89415 Create one resource file for each application

petershin committed:
LPS-89415 Update resource template

pavel-savinov committed:
LPS-89881 Use asset renderer to retrieve journal article

ealonso committed:
LPS-89154 Moves Editor Configuration

ealonso committed:
LPS-89154 Register for Content Page Editor Portlet

ealonso committed:
LPS-89154 Uses Content Page Editor Portlet to get editor configurations

ealonso committed:
LPS-89154 Moves common code to base class

ealonso committed:
LPS-89154 Adds className and classPK to constructor

ealonso committed:
LPS-89154 Moves getPanelSoyContexts to Base class

ealonso committed:
LPS-89154 Update actions

ealonso committed:
LPS-89154 Update urls on cards

pavel-savinov committed:
LPS-89154 Removes files

ealonso committed:
LPS-89154 Extract getSoyContextFragmentEntryLinks

ealonso committed:
LPS-89154 Extract common context objects

ealonso committed:
LPS-89154 Rename to make consistent

pavel-savinov committed:
LPS-89154 Implement fragments editor display context provider

ealonso committed:
LPS-89154 Don't use == to compare strings

ealonso committed:
LPS-89154 SF

pavel-savinov committed:
LPS-89154 Use fragments editor display context provider to create appropriate display context for layout and page templates editing

ealonso committed:
LPS-89154 Rename constant

ealonso committed:
LPS-89154 SF

Commit:
LPS-89423 Remove unused keys

4lejandrito committed:
LPS-88397 Registering factory to DLFileEntry

The classNameId being sent to the bulk API here:

	https://github.com/liferay/liferay-portal/blob/931611fab2b7e9768e99ec1cd70facb6c7eee1cd/modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view.jsp#L207

Comes from the DLFileEntry class:

	https://github.com/liferay/liferay-portal/blob/50888548989ba65c75f7cb7c9c90fa1957cd2117/portal-kernel/src/com/liferay/document/library/kernel/model/DLFileEntryConstants.java#L35

ealonso committed:
LPS-89154 Moves code to base class

Commit:
LPS-89423 Namespace special keys

Commit:
LPS-89423 Apply namespaces

Commit:
LPS-89423 Build lang

Commit:
LPS-89423 Remove organization count from User entity model

Commit:
LPS-89423 Add missing keys

davidavarga committed:
LPS-89071 Add data-qa-id

victorg1991 committed:
LPS-89723 Add dependencies

victorg1991 committed:
LPS-89723 Export

ealonso committed:
LPS-89503 Calculate it on display context

ealonso committed:
LPS-89899 Removes icon view

Commit:
LPS-89423 Fix param names

ealonso committed:
LPS-89899 Creates new SiteNavigationAdminManagementToolbarDisplayContext

petershin committed:
LPS-89415 Use Pagination

petershin committed:
LPS-89415 Parse YAML definitions

petershin committed:
LPS-89415 Update dto template

petershin committed:
LPS-89415 Remove debugging info

petershin committed:
LPS-89415 Regenerate

dgarciasarai committed:
LPS-89776 Remove GET collections paths for /content-space and /web-site from yaml

bryceosterhaus committed:
LRQA-44612 - Regen package-lock files

ealonso committed:
LPS-89899 Moves js actions to ManagementToolbarDefaultEventHandler

ealonso committed:
LPS-89154 Adds dependency

Commit:
LRQA-45896 Format source

Commit:
LRQA-45896 Rename batch names from open11 to jdk11_open

ealonso committed:
LPS-89899 Removes icon view

mtambara committed:
LPS-89640 Convert to URI to handle different platforms

brianchandotcom committed:
LPS-69035 prep next

brianchandotcom committed:
LPS-89415 Simplify

brianchandotcom committed:
LPS-89415 Add it everywhere

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/apio-architect/.gitrepo'.

brianchandotcom committed:
LPS-69035 prep next

nikki-pru committed:
LRQA-46524 Enable TensorFlow

petershin committed:
LPS-89415 Each schema should have its own resource

petershin committed:
LPS-89415 Update templates

petershin committed:
LPS-89415 Generated

brianchandotcom committed:
LPS-69035 new SDK

ealonso committed:
LPS-89899 Creates new SiteNavigationMenuItemSelectorViewManagementToolbarDisplayContext

brianchandotcom committed:
LPS-69035 Undo unnecessary changes

petershin committed:
Revert "LRQA-46521 Temporarily remove failing batch from relevant"

This reverts commit 53c656d155121e542e11cdf43d762f9bde375061.

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as c7450a46938401ac27a086af6d4cffb902106bc3

brianchandotcom committed:
Record reference to liferay-plugins/master with git-commit-plugins as 006066722ff86d58b127f2f29c8cd77cf9cabca2

kiyoshilee committed:
LRQA-46283 Modify the value for 'app.server.tomcat.version' when calling blades tests

brianchiu committed:
LPS-89604 Use card path to assert not present

brianchandotcom committed:
LPS-89615 Consistency

brianchandotcom committed:
LPS-89615 auto SF

brianchiu committed:
LPS-89604 Sort

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

brianchandotcom committed:
LPS-89154 Title

shuyangzhou committed:
LPS-89912 Case matters

brianchandotcom committed:
LPS-89154  Title

brianchandotcom committed:
LPS-89154 Regen

patriciajeanperez committed:
LRQA-46524 Auto SF

brianchandotcom committed:
LPS-88851 prep next

brianchandotcom committed:
LPS-89660 Wordsmith

thektan committed:
LPS-88934 Convert to a controlled input to keep value synced

kenjiheigel committed:
LRQA-46537 Update poshi runner to 1.0.214

brianchandotcom committed:
LPS-89375 SF

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 48077c702044a3d95d46dd65272df1eff2167fb9

brianchandotcom committed:
LPS-89415 Generator should no longer generate *Collection. Instead, use the common Page DTO

brianchandotcom committed:
LPS-89415 Add new Page DTO

brianchandotcom committed:
LPS-89415 Regenerate Workflow - this one works, I tested it

brianchandotcom committed:
LPS-89415 Remove Web Experience *Collection.java

brianchandotcom committed:
LPS-89415 Remove Web Experience .lfrbuild-portal because I can't run buildREST, I keep getting an error: - Failed at: ${parameter.schema.type?cap_first}  [in template "com/liferay/portal/tools/rest/builder/dependencies/resource.ftl" at line 160, column 57]

brianchandotcom committed:
LPS-89415 not used

brianchandotcom committed:
LPS-89776 add application config for foundation

brianchandotcom committed:
LPS-89723 add application config for collab

brianchandotcom committed:
LPS-89723 collab regen

brianchandotcom committed:
LPS-89711 Add application section to rest-config.yaml

brianchandotcom committed:
LPS-89711 add to form too

brianchandotcom committed:
LPS-88293 SF

brianchandotcom committed:
LPS-89415 SF

brianchandotcom committed:
LPS-89723 packageinfo

brianchandotcom committed:
auto SF

Commit:
LPS-89071 SF
